### DISCUSSION TITLE
Install is not clear (no vanilla css)

### DISCUSSION THREAD

**Discussion by br-ez** [sampotts/plyr#2556] [General]
Hi noob question,

I don't understand the install process listed for local linking. I have it working by using the CDN for the the css/js references (in an html file.)

But the download package doesn't have the usual 'dist' folder or seems to include a vanilla plyr.css as noted in the readme. There's just 'sass' files in the src folder for styling.

I'd like to self-host the 'plyr.css' vs CDN.

Understand this is a noob problem. Should I be compiling the sass into css or something using Visual Code to get that file format? Or just rip the css and js from the CDN as a download?

**Comment by sampotts** [comment]
A search would quickly find this, which should help: 
https://github.com/sampotts/plyr/discussions/2094
